stdlib/3.5/zipapp.pyi

```diff
@@ -0,0 +1,11 @@
+# Stubs for zipapp (Python 3.5+)
+
+from pathlib import Path
+from typing import BinaryIO, Optional, Union
+
+_Path = Union[str, Path]
```

Comment:
You are always unifying it with BinaryIO anyway so maybe it'd be easier to add it here?